examples/quickstart/jupyter-notebooks/nested-columns-tutorial.ipynb

```diff
@@ -57,28 +57,30 @@
    "source": [
     "## Prerequisites\n",
     "\n",
-    "You'll need install the Requests library for Python before you start. For example:\n",
+    "You need to install the requests library for Python before you start&mdash;for example:\n",
```

Comment:
The official name of the library uses `R` not `r`. It's kind of hard to notice on t heir docs/repo cause they keep putting it at the start of a sentence

Reply:
Updated